sdk/storage/Azure.Storage.Blobs/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("ApiDesign", "AZC0035:Output model type should have a corresponding method in a model factory class", Justification = "This type has a public constructor and all properties are settable.", Scope = "type", Target = "~T:Azure.Storage.Blobs.Models.BlobServiceProperties")]
```

Comment:
check which of these exclusions are still needed given the new analyzer package.